docs/get-started.md

```diff
@@ -107,160 +107,143 @@ Welcome to Nebula Graph (Version 0.1)
 ```
-CREATE space myspace(partition_num=1, replica_factor=1)
+The **CREATE EDGE** statement defines an edge type.
+```shell
+nebula> CREATE EDGE like(likeness double);
+nebula> CREATE EDGE select(grade int);
 ```
 
-When you start `Nebula Graph`, a set of vertices and edges have already existed.
-Currently the default schema is `nba`.
+To list the tags and edge types we just created：
+```shell
+// show tag list
+nebula> SHOW TAGS;
 
-The graph space describes the relationship between players and teams.
-In the graph space, there are two tags (player and team) and two edges (serve and like) which is composed of `string` and `integer`.
+// show edge type list
+nebula> SHOW EDGES;
+```
 
-The Schema looks like :
+To show the attributes of a tag or an edge type:
+```shell
+// show attributes of a tag
+nebula> DESCRIBE TAG student;
 
+//show attributes of an edge type 
+nebula> DESCRIBE EDGE like;
 ```
-Tag team:
-==================
-| Field |   Type |
-==================
-|  name | string |
-------------------
-
-Tag player:
-==================
-| Field |   Type |
-==================
-|  name | string |
-------------------
-|   age |    int |
-------------------
-
-Edge serve:
-=====================
-|      Field | Type |
-=====================
-| start_year |  int |
----------------------
-|   end_year |  int |
----------------------
-
-Edge like:
-===================
-|    Field | Type |
-===================
-| likeness |  int |
--------------------
 
+#### Insert Data
+Insert the vertexes and edges based on the graph above.
+```shell
+//insert vertexes
+nebula> INSERT VERTEX student(name, age, gender) VALUES 200:("Monica", 16, "female");
+nebula> INSERT VERTEX student(name, age, gender) VALUES 201:("Mike", 18, "male");
+nebula> INSERT VERTEX student(name, age, gender) VALUES 202:("Jane", 17, "female");
+nebula> INSERT VERTEX course(name, credits),building(name) VALUES 101:("Math", 3, "No5");
+nebula> INSERT VERTEX course(name, credits),building(name) VALUES 102:("English", 6, "No11");
+
+//insert edges
+nebula> INSERT EDGE select(grade) VALUES 200 -> 101:(5);
+nebula> INSERT EDGE select(grade) VALUES 200 -> 102:(3);
+nebula> INSERT EDGE select(grade) VALUES 201 -> 102:(3);
+nebula> INSERT EDGE select(grade) VALUES 202 -> 102:(3);
+nebula> INSERT EDGE like(likeness) VALUES 200 -> 201:(92.5);
+nebula> INSERT EDGE like(likeness) VALUES 201 -> 200:(85.6);
+nebula> INSERT EDGE like(likeness) VALUES 201 -> 202:(93.2);
 ```
 
-You can create both tag and edge's schema:
+#### Sample Queries
+Q1. Find the vertexes that 201 likes:
+```shell
+nebula> GO FROM 201 OVER like;
 
+=======
+|  id |
+=======
+| 200 |
+-------
+| 202 |
+-------
 ```
-// create tags schema: players and teams:
-CREATE TAG player(name string, age int)
-
-CREATE TAG team(name string)
+Q2. Find the vertexes that 201 likes, whose age are greater than 17. Return their name, age and gender, and alias the columns as Friend, Age and Gender, respectively.
```

Comment:
alias ( or rename)

Reply:
https://www.quora.com/What-is-the-difference-between-rename-and-alias-in-DBMS-1

Reply:
https://www.quora.com/What-is-the-difference-between-rename-and-alias-in-DBMS-1 😝😝